File: jdi tests/jdi-uitest-tutorialtests/src/main/java/com/epam/jdi/uitests/testing/career/page_objects/site/pages/JobDescriptionPage.java
package com.epam.jdi.uitests.testing.career.page_objects.site.pages;

import com.epam.jdi.uitests.core.interfaces.base.IElement;
import com.epam.jdi.uitests.testing.career.page_objects.site.sections.AddCVForm;
import com.epam.jdi.uitests.web.selenium.elements.composite.WebPage;
import org.openqa.selenium.support.FindBy;

/**
 * Created by Roman_Iovlev on 10/22/2015.
 */
public class JobDescriptionPage extends WebPage {

    @FindBy(css = ".form-constructor")
    public AddCVForm addCVForm;

    @FindBy(id = "captcha-input")
    public IElement captcha;

}

File: C# .Net/JDI UI Framework/Tests/Scenarios/Page Objects/Pages/JobDescriptionPage.cs
using Epam.JDI.Core.Interfaces.Base;
using Epam.JDI.Web.Attributes;
using Epam.JDI.Web.Selenium.Elements.Composite;
using Epam.Tests.Scenarios.Page_Objects.Sections;

namespace Epam.Tests.Scenarios.Page_Objects.Pages
{
    public class JobDescriptionPage : WebPage
    {
        [FindBy(Css = ".form-constructor")]
        public AddCVForm AddCvForm;

        [FindBy(Id = "captcha-input")]
        public IElement Captcha;
    }
}
